### PR TITLE
Add video demo

### PR DESCRIPTION
**Pull request by zoek1**
##### Description
- Add video field to submission view
- Add field to project form
- Show video demo in the project page

https://www.loom.com/share/b6f604db4a854427a42a85cb88b71939

### CODE REVIEW

**Comment by thelostone-mc** [comment]
Could you list out which issue this is linked to?

**Review by PixelantDesign — APPROVED** [review]

**Comment by PixelantDesign** [comment]
awesome!

**Comment by PixelantDesign** [comment]
Looking good! @zoek1 if @jdorfman or @connoroday set up the event on  Add Event, the events will show on our events tab correct? I didn't see that in the video.

**Comment by frankchen07** [comment]
> set up the event on Add Event, the events will show on our events tab correct? I didn't see that in the video.

yes this was part of the other ticket: https://www.loom.com/share/a82ef2081ffb45e6bef233fe9214112d

it looks like as long as the calendar id is linked in the administration, using add event will show up.

===

just want to rehash adityas request above to link the issue ticket this refers to.

otherwise LGTM, looking good. Hopefully this helps sponsors review hackathon work at a greater pace, and allows folks to have a pre-recorded pitch.

**Review by frankchen07 — APPROVED** [review]

**Comment by PixelantDesign** [comment]
yes! i dropped comment on wrong ticket  and figured out after rewatched lol

File: app/assets/v2/js/vue-components.js [hunk 1]
```diff
@@ -527,23 +527,6 @@ Vue.component('project-card', {
...
-      }, {'X-CSRFToken': vm.csrf});
-
-      $.when(markWinner).then(response => {
-        if (response.message) {
-          alert(response.message);
-        }
-      }).catch(err => {
-        console.log(err);
-      });
-    },
```
Comment (octavioamu) [inline app/assets/v2/js/vue-components.js:546]: is this removal related?

**Review by octavioamu — CHANGES_REQUESTED** [review]
seems some other code come in to remove the mark winner

File: app/assets/v2/js/vue-components.js (hunk 1 above)
Comment (octavioamu) [reply app/assets/v2/js/vue-components.js:546]: As I didn't saw a comment about it on the description I requested changes. Not sure if product team is aware of that changes also.

**Comment by PixelantDesign** [comment]
https://github.com/gitcoinco/web/issues/7507

**Comment by PixelantDesign** [comment]
>seems some other code come in to remove the mark winner
@octavioamu what do you mean here?

**Comment by zoek1** [comment]
This is included in https://github.com/gitcoinco/web/pull/7493

**Comment by PixelantDesign** [comment]
@thelostone-mc can we get this pr in first?

**Pull request closed without merging**